go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/pkg/term v0.0.0-20180730021639-bffc007b7fd5 // indirect
 	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
 	github.com/prometheus/common v0.6.0
-	github.com/satori/go.uuid v1.2.0
+	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
```

Comment:
Can we get them to do a release for this? It also looks like a breaking change so I am not sure how they want to handle this.

Reply:
@jsternberg "them" as in the compute team? Or who?

Reply:
Whoever manages this dependency. The reason why I'm concerned about this is this could break downstream projects, such as influxdb, that use the older dependency and break their API because it forces them to update. This isn't that big of a deal right now, but I would prefer a numbered version rather than a random hash.

Reply:
I created a new issue for tracking this work #2392